dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -96,6 +108,8 @@ KEYCLOAK_GITHUB_ENDPOINT=${KEYCLOAK_GITHUB_ENDPOINT:-${DEFAULT_KEYCLOAK_GITHUB_E
 DEFAULT_OPENSHIFT_FLAVOR=minishift
 OPENSHIFT_FLAVOR=${OPENSHIFT_FLAVOR:-${DEFAULT_OPENSHIFT_FLAVOR}}
 
+#TODO will probably go to the main config map at then end
```

Comment:
Please be more specific so that someone that is not @davidfestal can understand :-). Something like (not sure I've understand what need to be done but...):
```bash
# TODO move this env variable as a config map in the deployment config as soon as che-multiuser branch is merged to master
```